listenbrainz/db/user.py

```diff
@@ -117,7 +118,7 @@ def get_by_token(token):
     """
     with db.engine.connect() as connection:
         result = connection.execute(sqlalchemy.text("""
-            SELECT id, created, musicbrainz_id
+            SELECT id, created, musicbrainz_id, last_login
```

Comment:
I'm just glancing at this duplicated code 3× in the same file and wondering if we could have a `USER_SELECT_FIELDS` variable or something which lists these fields